eip-draft_clwp.md

```diff
@@ -0,0 +1,202 @@
+---
+eip: <to be assigned>
+title: Consensus Layer Withdrawal Protection
+description: Provides additional security in the "change withdrawal address" operation when a consensus layer mnemonic may be compromised, without changing consensus
+author: Benjamin Chodroff (@benjaminchodroff), Jim McDonald (@mcdee)
+discussions-to: https://ethereum-magicians.org/t/consensus-layer-withdrawal-protection/8161
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): Interface
+created: 2022-01-30
+requires (*optional): <EIP number(s)>
```

Comment:
```suggestion
category: Interface
created: 2022-01-30
```

Can drop the `requires` section, and the stuff in parentheses is just a note for authors.